src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3334,47 +3334,84 @@
+  "botania.subtitle.gaiaSummon": "Gaia Guardian awakens",
+  "botania.subtitle.gaiaTeleport": "Gaia Guardian teleports",
   "botania.subtitle.gaiaTrap": "Gaia Guardian's spell swooshes",
-  "botania.subtitle.holyCloak": "Cloak of Virtue activates",
-  "botania.subtitle.labellia": "Labellia renames something",
+  "botania.subtitle.holyCloak": "Cloak of Virtue shines",
+  "botania.subtitle.hornDoot": "Horn doots",
+  "botania.subtitle.labellia": "Labellia emblazons",
   "botania.subtitle.laputaStart": "Laputa Shard triggers",
   "botania.subtitle.lexiconOpen": "Lexica Botania opens",
   "botania.subtitle.lexiconPage": "Lexica Botania page turns",
   "botania.subtitle.lightRelay": "Luminizer whooshes",
   "botania.subtitle.manaBlaster": "Mana Blaster fires",
+  "botania.subtitle.manaBlasterCycle": "Lens Clip whirrs",
+  "botania.subtitle.manaBlasterMisfire": "Mana Blaster misfires",
   "botania.subtitle.manaPoolCraft": "Mana Pool infuses",
   "botania.subtitle.missile": "Magic Missile fires",
-  "botania.subtitle.orechid": "Stone transforms into ore",
-  "botania.subtitle.potionCreate": "Brewery finishes",
-  "botania.subtitle.runeAltarCraft": "Runic Altar finishes",
+	"botania.subtitle.missileFunny": "pew pew",
+  "botania.subtitle.narslimmusEat": "Narslimmus squishes",
+  "botania.subtitle.orechid": "Orechid bangs",
+  "botania.subtitle.pinkinator": "Pinkinator pinkinates",
+  "botania.subtitle.potionCreate": "Brewery crackles",
+  "botania.subtitle.redStringInterceptorClick": "Red Stringed Interceptor clicks",
+  "botania.subtitle.runeAltarCraft": "Runic Altar crackles",
   "botania.subtitle.runeAltarStart": "Runic Altar starts",
-  "botania.subtitle.spreaderFire": "Spreader shoots mana",
+  "botania.subtitle.shulkMeNot": "Shulk Me Not bangs",
+  "botania.subtitle.smeltRod": "Rod of the Molten Core cooks",
+  "botania.subtitle.smeltRodSimmer": "Rod of the Molten Core simmers",
+  "botania.subtitle.spreaderCover": "Spreader is covered",
+  "botania.subtitle.spreaderFire": "Spreader fwooshes",
+  "botania.subtitle.spreaderUncover": "Spreader is uncovered",
   "botania.subtitle.starcaller": "Stars descend from the sky",
+  "botania.subtitle.temperanceStoneConfigure": "Stone of Temperance dings",
   "botania.subtitle.terraBlade": "Terrasteel Sword activates",
-  "botania.subtitle.terraformRod": "Land terraforms",
   "botania.subtitle.terraPickMode": "Terrasteel Pick powers up",
-  "botania.subtitle.terrasteelCraft": "Terrasteel finishes",
-  "botania.subtitle.thermalily": "Thermalily consumes lava",
+  "botania.subtitle.terraformRod": "Land terraforms",
+  "botania.subtitle.terrasteelCraft": "Terrasteel shines",
+  "botania.subtitle.thermalily": "Thermalily sips",
+  "botania.subtitle.thornChakramThrow": "Thorn Chakram thrown",
+  "botania.subtitle.tigerseyePacify": "Tigerseye petrifies",
   "botania.subtitle.unholyCloak": "Cloak of Sin activates",
+  "botania.subtitle.vineBallThrow": "Vine Ball flies",
```

Comment:
seems a bit weird, maybe just `is thrown` like ender air?

Reply:
took this wording from snowball subtitle i think